metricbeat/module/system/diskio/_meta/fields.yml

```diff
@@ -54,3 +54,58 @@
       type: long
       description: >
         The total number of of milliseconds spent doing I/Os.
+
+    - name: iostat.rrqmCps
```

Comment:
I would vote for the long version. Reasons is that this will allow us to group some of the events into namespaces and use `units` like `.sec` which are in line with the other events.

I tried my google skills on the abbreviations below but mostly ended up on this discuss issue: https://discuss.elastic.co/t/why-system-diskio-doesnt-produce-i-o-statistics-fields-like-iostat/91354/3